rasa/core/interpreter.py

```diff
@@ -37,7 +37,13 @@ def create(
         if isinstance(obj, NaturalLanguageInterpreter):
             return obj
         elif isinstance(obj, str) and os.path.exists(obj):
-            return RasaNLUInterpreter(model_directory=obj)
+            from rasa.nlu.model import UnsupportedModelError
```

Comment:
What do you think of adding the exception handler [here](https://github.com/RasaHQ/rasa/blob/cd1500ec5c1cdb9b1dc87a7e91edb408966c296e/rasa/core/run.py#L249) ? That way the change is really isolated to the startup and we handle the error as one, and not separate for Core and NLU?

Let's also add a test for it 👍

Reply:
Good idea, since these model-loading fatal errors only occur at boot. I have since also gotten `zlib.error` exceptions (from corrupt tarballs). So I think a fully general exception catching block is in order? cf. https://github.com/RasaHQ/rasa/blob/d52d50625676cc679ec83dcd4e663429f7ce2d73/rasa/server.py#L315-L322 Will submit revision with test. 🚀